test/test_common.py

```diff
@@ -88,6 +90,40 @@ def test_json_serialization(self, tmpdir):
 
         assert ob2 == ob2_
 
+    def test_persisted_progress(self):
+        test_progress_file = "test_progress.json"
+
+        test_data = {
+            "progress": {
+                "0af668268d0fe14c6e269760b08d80a634c421b8381df25f31fbed5e8a8c8d8b": {
+                    "current_progress": 100,
+                    "last_updated_time": 1625072042.123456
+                }
+            }
+        }
+
+        with open(test_progress_file, "w") as file:
+            json.dump(test_data, file)
+
+        with patch("pghoard.common.PROGRESS_FILE", test_progress_file):
```

Comment:
you can use the mocker fixture to patch the constant that way you don't need this extra level of indentation